examples/demo.md

```diff
@@ -153,6 +153,230 @@ done
 
 <!-- end_slide -->
 
+Shell Rust code execution 1
```

Comment:
Can you remove this from the demo presentation? This is meant to be a very minimal presentation to showcase how the tool works only.